environment_setup/setup_software.sh

```diff
@@ -215,6 +215,22 @@ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 \
                          --slave /usr/bin/g++ g++ /usr/bin/g++-7 
 sudo update-alternatives --config gcc
 
+# Clone, build, and install g3log. Adapted from instructions at:
+# https://github.com/KjellKod/g3log
+g3log_path="/tmp/g3log"
+if [ -d $g3log_path ]; then
+    echo "Removing old g3log..."
+    sudo rm -r $g3log_path
+fi
+
+git clone https://github.com/KjellKod/g3log.git $g3log_path
```

Comment:
Should the URL be some kind of constant string at the top of the file?

Reply:
Don't think so. All our other scripts keep the urls and variables with the sections that use them. Leaving this as it is.